site/assets/scss/_component-examples.scss

```diff
@@ -99,7 +99,7 @@
   }
 
   + .highlight,
-  + .clipboard + .highlight {
+  + .bd-clipboard + .highlight {
     margin-top: 0;
   }
```

Comment:
The `application.js` insert a `div.clipboard` block before every .`div.highlight` block. So the following `margin-top: 1rem` can be removed:

https://github.com/twbs/bootstrap/blob/36a68b1bc0482c3258d4fd81eb144a36e298b2e8/site/assets/scss/_component-examples.scss#L242-L244

Therefore, no need to override the `margin-top: 0` (L101-104) and the below too:

https://github.com/twbs/bootstrap/blob/36a68b1bc0482c3258d4fd81eb144a36e298b2e8/site/assets/scss/_clipboard-js.scss#L5-L12